zork++/src/lib/cli/mod.rs

```diff
@@ -30,49 +29,53 @@ use clap::{Parser, Subcommand, ValueEnum};
 )]
 pub struct CliArgs {
     #[command(subcommand)]
-    pub command: Option<Command>,
+    pub command: Command,
 
     #[arg(short, long, action = clap::ArgAction::Count, help="Zork++ maximum allowed verbosity level is: '-vv'")]
     pub verbose: u8,
-
-    #[arg(short, long, group = "base_option", help = "Create Project template")]
-    pub new_template: bool,
-    
-    #[arg(
-        long,
-        help = "Initializes a new local git repo"
-    )]
-    pub git: bool,
-    
-    #[arg(
-        long,
-        help = "Indicates what compiler wants the user to use with the autogenerated project"
-    )]
-    pub compiler: Option<CppCompiler>,
 }
 
 /// [`Command`] -  The core enum commands
 #[derive(Subcommand, Debug, PartialEq, Eq)]
 pub enum Command {
     /// Executes the tests under the specified directory in the config file
-    Tests,
+    Test,
```

Comment:
Command line for the project is starting to have useful meaning for the end user. Good point on this